Express.js required on GCF

Sorry in advance, this is my first time using GCF.

Seems like it wants to run the `start` script automatically. But now that #3 is in, `start` points to `local.js`, which requires express.js.

I'll attach a PR shortly that separates out `start` from `start-dev`. But if there's a better way to do this that's specific to GCF, that would be better.

No PR - making the change didn't work. GCF still complains that it can't load local.js . What am I missing?

log output:

```
Deployment failure:

Function load error: Code in file local.js can't be loaded.
Did you list all required modules in the package.json dependencies?
Detailed stack trace: Error: Cannot find module 'express'
    at Function.Module._resolveFilename (module.js:547:15)
    at Function.Module._load (module.js:474:25)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/srv/local.js:1:79)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
```

--- a/README.md
+++ b/README.md
@@ -33,7 +33,7 @@ For YNAB:
 
 `ENABLE_SCREENSHOTS`: for debugging
 
-Then, you can run a local version by `yarn start` and visiting http://localhost:3000 for triggering the function.
+Then, you can run a local version by `yarn start-dev` and visiting http://localhost:3000 for triggering the function.
 
 ### Cloud Functions
 - Setup a new Cloud Function with:

--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "db-to-ynab",
   "version": "1.0.0",
-  "main": "local.js",
+  "main": "index.js",
   "license": "MIT",
   "scripts": {
     "start": "node index.js",

--- a/package.json
+++ b/package.json
@@ -4,7 +4,8 @@
   "main": "local.js",
   "license": "MIT",
   "scripts": {
-    "start": "node local.js",
+    "start": "node index.js",
+    "start-dev": "node local.js",
     "lint": "yarn eslint ."
   },
   "dependencies": {

